### PR TITLE
fix KDE tray icon issue

### LINKED ISSUE #7: Icon is missing in a system tray in KDE
Hi!

I've installed the latest Gitter from Flathub, but instead of real Gitter icon in a system tray I see blank icon.

OS: Fedora 31 KDE
KDE Frameworks: 5.68.0
KDE Plasma: 5.17.5
Qt: 5.13.2
Flatpak 1.4.4

### PR DESCRIPTION
**Pull request by ctron**
fixes #7

### CODE REVIEW

**Comment by ghisvail** [comment]
Could you post a screenshot with the working KDE notification?

**Comment by ctron** [comment]
image

File: im.gitter.Gitter.yaml [hunk 1]
```diff
@@ -10,8 +10,14 @@ finish-args:
   - --device=dri
   - --share=ipc
   - --socket=x11
```
Comment (ghisvail) [inline im.gitter.Gitter.yaml:12]: You can remove the x11 socket since fallback-x11 overrides it.

Comment (ghisvail) [inline im.gitter.Gitter.yaml]: You can remove the x11 socket since fallback-x11 overrides it.

**Review by ghisvail — CHANGES_REQUESTED** [review]

**Comment by ghisvail** [comment]
Thanks for your work

**Pull request merged**